clippy_lints/src/methods/mod.rs

```diff
@@ -2348,8 +2348,8 @@ fn lint_iter_nth_zero<'tcx>(cx: &LateContext<'tcx>, expr: &hir::Expr<'_>, nth_ar
                 cx,
                 ITER_NTH_ZERO,
                 expr.span,
-                "called `.nth(0)` on a `std::iter::Iterator`",
-                "try calling",
+                "called `.nth(0)` on a `std::iter::Iterator`, when `.next()` is equivalent",
+                "try calling .next() instead of .nth(0)",
```

Comment:
NIT:

```suggestion
                "try calling `.next()` instead of `.nth(0)`",
```

(remember to also update the stderr files after this change)